src/kernel/region.c

```diff
@@ -624,10 +628,28 @@ int rhorses(const region * r)
 
 void rsetmoney(region * r, int value)
 {
-    if (r->land) r->land->money = value;
-    else assert(value >= 0);
+    if (r->land) {
+        assert(value >= 0);
+        r->land->money = value;
+    } else
+        assert(value == 0);
 }
 
+short rherbs(const struct region *r)
```

Comment:
Es ist generell einfacher, wenn Funktionen keine Typen wie short oder float zurück liefern, das sind Typen, die maximal in einer struct benutzt werden, um Speicherplatz niedrig zu halten, aber für allgemeine Berechnungen ist es einfacher, wenn alles in ist und Double abläuft, finde ich.